examples/06_benchmarks/movielens.ipynb

```diff
@@ -72,28 +72,39 @@
    "execution_count": 1,
```

Comment:
<p>The warnings are annoying, you can get rid of them by adding:</p><pre class="ql-syntax" spellcheck="false">import warnings
warnings.filterwarnings("ignore")

</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>